README.adoc

```diff
@@ -0,0 +1,163 @@
+  discussed in that PR and/or the RFC. The discussion may incur doing changes
+  to the RFC.
+
+. After sufficient time and discussion the combined RFC and wording PR will
+  be considered ready for a decision on further progress. A member of the
+  "adoption review group" will propose taking a vote for adoption.
+
+. An adoption vote starts after three members of the review group agree. The
+  vote will last at most 28 calendar days, and at least 7 calendar days. The
+  length of the voting period will be part of the voting proposal. But the
+  length is subject to community feedback before and during the voting. The
+  RFC PR will be labeled accordingly to reflect the voting.
+
+. When the adoption vote completes the RFC is either approved or rejected. The
+  RFC will be labeled accordingly and may be closed.
+
+. A rejected RFC PR may be closed depending on the reasons for rejection. The
+  RFC may stay active, but go back to being discussed, if there are changes
+  possible and reconsidered for approval.
+
+. An approved RFC PR is closed, and labeled as adopted. The actions proscribed
+  in the RFC are then taken up by others to complete. For RFCs that affect the
+  https://github.com/ecostd/standard[standard] will continue in the related
+  PR by the "standard editors group".
+
+TIP: The preferred document format for RFCs is
+https://asciidoctor.org[Asciidoctor]. But the
+https://github.github.com/gfm/[GitHub Flavored Markdown] format is supported as
+an alternative. The extent of support is subject to what is supported by the
+conversion to Asciidoctor process as part of build the HTML for the
+https://ecostd.github.io/rfcs/[RFCs website].
+
+[#guidelines]
+== Guidelines
+
+These are a loose collection of rules to follow, things to keep in mind, and
+background information that is useful when creating RFCs. They are grouped by
+the aspect of RFCs that they apply to:
+
+<<guidelines-mechanics>>:: Guidelines for the task of the RFC and PR itself.
+<<guidelines-design>>:: Guidelines for that help with the design content of an RFC.
+
+These are _guidelines_, and not _rules_. They are expected to be true most
+times. But we allow for flexibility in the process. The goal is to reduce
+friction as much as possible without sacrificing comprehensive evaluation of
+RFCs.
+
+[#guidelines-mechanics]
+=== Mechanics
+
+Do not squash, or otherwise coalesce, changes for RFC PRs::
+After creating an RFC PR additional updates should be done as additional, plain,
+commits. Having the additional commits helps in tracking how feedback is
+addressed. When an RFC is approved it will be squash merged. Hence attempts to
+pre-squash would be wasted.
+
+Mirror external feedback in the PR::
+Any feedback you get outside of PR comments should be summarized, by you, as
+comments in the PR. And if there are changes, i.e. additional commits, to the
+RFC that are related they should be mentioned in the PR comments.
+
+Do not change approved PRs::
```

Comment:
What about an approved branch, so as soon as something is approved, it's merged into approved and can not be changed anymore.  That should be possible to automate via CI / gh actions